app/src/main/java/org/liberty/android/fantastischmemo/common/AnyMemoDBOpenHelper.java

```diff
@@ -199,6 +199,11 @@ public void onUpgrade(SQLiteDatabase database, ConnectionSource connectionSource
         if(oldVersion <= 7){
             database.execSQL(CREATE_MULTIPLE_CHOICE_TABLE);
         }
+
+        if(oldVersion <= 8){
+            database.execSQL("alter table settings add hintAudio String");
```

Comment:
What are the values that both of the columns are holding? Why do we need two?

Reply:
hintAudio is the actual audio file, and hintAudioLocation is the path where all the audio files are located.